bindings/python/unicorn/unicorn.py

```diff
@@ -202,6 +202,14 @@ class uc_x86_float80(ctypes.Structure):
     ]
 
 
+class uc_x86_xmm(ctypes.Structure):
+    """128-bit xmm register"""
+    _fields_ = [
+        ("lowdword", ctypes.c_uint64),
```

Comment:
Oh, dang, this completely dropped off my radar. I'm fine with that, let me make the change real quick.